docs/reference/docs/reindex.asciidoc

```diff
@@ -136,7 +136,7 @@ POST _reindex
 // TEST[setup:twitter]
 
 You can limit the documents by adding a type to the `source` or by adding a
-query. This will only copy ++tweet++&apos;s made by `kimchy` into `new_twitter`:
+query. This will only copy ++tweet++s made by `kimchy` into `new_twitter`:
```

Comment:
I wonder if you could close #29231 and use this PR for those changes instead.

Reply:
All set

Reply:
Forgot about that. Fixed!